data/model/team.py

```diff
@@ -608,14 +609,22 @@ def get_oidc_team_from_groupname(group_name, login_service_name):
     Fetch TeamSync row synced with login_service_name from `group_name` in TeamSync.config
     """
     response = []
-    with db_transaction():
-        query_result = (
-            TeamSync.select()
-            .join(LoginService)
-            .where(TeamSync.config.contains(group_name), LoginService.name == login_service_name)
-        )
-        for row in query_result:
-            if json.loads(row.config).get("group_name", None) == group_name:
-                response.append(row)
+    query_result = (
+        TeamSync.select()
+        .join(LoginService)
+        .where(TeamSync.config.contains(group_name), LoginService.name == login_service_name)
+    )
+
+    for row in query_result:
+        if json.loads(row.config).get("group_name", None) == group_name:
+            response.append(row)
 
     return response
+
+
+def user_exists_in_team(user_obj, team):
+    try:
+        TeamMember.get(TeamMember.user == user_obj, TeamMember.team == team)
+        return True
+    except TeamMember.DoesNotExist:
+        return False
```

Comment:
nit: doesn't need to be fixed now but this can just be `return TeamMember.select().where(TeamMember.user == user_obj, TeamMember.team == team).exists()`

Reply:
nice! will update